### PR TITLE
Add New Featured Artist: Darkney newspost

### PR DESCRIPTION
**Pull request by Ephemeralis**
🎵 

(A smidge late for *reasons*.)

### CODE REVIEW

File: news/2022-08-06-new-featured-artist-darkney.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+date: 2022-08-06 18:00:00 +0000
+---
+
+Step into the light with our newest Featured Artist, **Darkney**!
+
+![](https://assets.ppy.sh/artists/290/header.jpg)
+
+Here at osu!, we're lucky to be the beneficiary of many different types of musical material from a variety of artists. First we had image material, and now courtesy of [**Darkney**](https://osu.ppy.sh/beatmaps/artists/290), we've even moved into [*Cloud Material*](https://osu.ppy.sh/beatmapsets/739583) as well.
+
+Those familiar with the vintage 2018 [Monstrata](https://osu.ppy.sh/users/2706438) classic linked above will be excited to find out that we've snagged **13** new tracks from the talented German producer responsible for their creation. If you're a fan of his uniquely atmospheric spin on future brass x EDM (and we think you will be), this is a grand day indeed.
```
Comment (Walavouchey) [inline news/2022-08-06-new-featured-artist-darkney.md:13]:
```suggestion
Those familiar with the vintage 2018 [Monstrata](https://osu.ppy.sh/users/2706438) classic linked above will be excited to find out that we've snagged **13** new tracks from the talented German producer responsible for their creation. If you're a fan of his uniquely atmospheric spin on future bass x EDM (and we think you will be), this is a grand day indeed.
```

File: news/2022-08-06-new-featured-artist-darkney.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+If you just can't quite put your finger on what the hype is all about, don't worry, [Bubbleman](https://osu.ppy.sh/users/5182050) has quite literally done it for you in the video below at during an IRL performance at COE2022, so watch along and see for yourself:
```
Comment (Walavouchey) [inline news/2022-08-06-new-featured-artist-darkney.md:15]:
```suggestion
If you just can't quite put your finger on what the hype is all about, don't worry, [Bubbleman](https://osu.ppy.sh/users/5182050) has quite literally done it for you in the video below during an IRL performance at COE2022, so watch along and see for yourself:
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: news/2022-08-06-new-featured-artist-darkney.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+layout: post
+title: "New Featured Artist: Darkney"
+date: 2022-08-06 18:00:00 +0000
+---
+
+Step into the light with our newest Featured Artist, **Darkney**!
+
+![](https://assets.ppy.sh/artists/290/header.jpg)
+
+Here at osu!, we're lucky to be the beneficiary of many different types of musical material from a variety of artists. First we had image material, and now courtesy of [**Darkney**](https://osu.ppy.sh/beatmaps/artists/290), we've even moved into [*Cloud Material*](https://osu.ppy.sh/beatmapsets/739583) as well.
```
Comment (Walavouchey) [inline news/2022-08-06-new-featured-artist-darkney.md:11]: is this reference supposed to be subtle? if not then i suggest linking the map

Comment (Ephemeralis) [reply news/2022-08-06-new-featured-artist-darkney.md:11]: yes, also i don't want to link an artist-unrelated map inside an FA article, most readers will have heard of image material before i'm sure

**Review by peppy — APPROVED** [review]

**Pull request merged**